grammars/grammars.json

```diff
@@ -351,8 +351,15 @@
 		{
 			"language": "stylus",
 			"scopeName": "source.stylus",
-			"extensions": [".stylus"],
-			"grammar": "Stylus.tmLanguage"
+			"extensions": [".stylus", ".styl"],
+			"configuration": {
+				"base": "stylus",
```

Comment:
Is there a base stylus grammar on Monaco? Can't recall exactly but I think that's what this is meant for?

Reply:
This `configuration` object resolves to a path, `"stylus/language-configuration.json"`. If it's not specified, we infer a default path from the `language`, as `${language}/language-configuration.json`.

I’ve taken to make the `configuration` and `grammar` paths more explicit instead of convention-based, mostly because I like when stuff is explicit ^^